static/app/views/explore/hooks/useTraceItemDetails.tsx

```diff
@@ -91,11 +92,12 @@ export type TraceItemResponseAttribute =
  */
 export function useTraceItemDetails(props: UseTraceItemDetailsProps) {
   const organization = useOrganization();
+  const {fetching} = useProjects();
   const project = useProjectFromId({project_id: props.projectId});
   const enabled = (props.enabled ?? true) && !!project;
```

Comment:
Enabled already doesn't occur until `project` is loaded, so the capture exception below is only in case where project has failed to load despite everything loading.